app/Mail/ResourceDigestEmail.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class ResourceDigestEmail extends Mailable implements ShouldQueue
+{
+    use Queueable, SerializesModels;
+
+    public $resources;
+    public $user;
+    public $unsubscribeUrl;
+
+    public function __construct($resources, $user, $unsubscribeUrl)
+    {
+        $this->resources = $resources;
+        $this->user = $user;
+        $this->unsubscribeUrl = $unsubscribeUrl;
+    }
```

Comment:
This can be rewritten like the following and then the properties on lines 16-18 can be removed.

```suggestion
    public function __construct(
        public $resources,
        public $user,
        public $unsubscribeUrl
    ) {}
```